Smith_Ben/Assignments/Basic Python Fund/multiply.py

```diff
@@ -0,0 +1,8 @@
+a = [2, 4, 10, 16]
```

Comment:
what happens if I redefine a = [1,2,3,4,5,6,7,8,9] ? the multiply function fails because it only goes to range(4). You should alter it so that it traverses the range of the len(a), _--OR--_, alternatively, you can iterate the elements of a directly, e.g.: `for ele in x: ele*=5`